CodeCoverageQualityGate/package.json

```diff
@@ -21,5 +21,16 @@
   "bugs": {
     "url": "https://github.com/parasoft/parasoft-findings-vsts/issues"
   },
-  "homepage": "https://github.com/parasoft/parasoft-findings-vsts#readme"
+  "homepage": "https://github.com/parasoft/parasoft-findings-vsts#readme",
+  "devDependencies": {
+    "@types/q": "^1.5.5",
+    "@types/node-fetch": "^2.6.5",
+    "license-checker-rseidelsohn": "^4.2.5"
```

Comment:
The library for generating licenses has been added, but the corresponding npm run script has not. I believe it is also a good practice to include the license in the same pull request whenever a new dependency is added. Therefore, please add a script for running the npm command to generate licenses and create a licenses folder.